src/components/Farming/DepositTable/index.js

```diff
@@ -103,24 +105,32 @@ const DepositTable = props => {
           setRewards,
           symbol,
           rewardBalance,
+          pairInfo,
         }) => {
           const imgObj = {
-            pair1: getKnownTokenImg(symbol),
-            pair2: null,
+            pair1: pairInfo ? pairInfo.token0.logoURI : undefined,
```

Comment:
same than previous comment, shouldn't use the default image instead of undefined?